content/actions/using-workflows/workflow-commands-for-github-actions.md

```diff
@@ -733,7 +733,7 @@ steps:
 {% ifversion actions-save-state-set-output-envs %}
 ## Setting an output parameter
 
-Sets a step's output parameter. Note that the step will need an `id` to be defined to later retrieve the output value.
+Sets a step's output parameter. Note that the step will need an `id` to be defined to later retrieve the output value. Multi-line output values can be set with the same technique used in the [Multiline strings](/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings) section to define multi-line environment variables.
```

Comment:
Used double quotes for the link and the active voice (to be in line with our content model)
```suggestion
Sets a step's output parameter. Note that the step will need an `id` to be defined to later retrieve the output value. You can set multi-line output values with the same technique used in the "[Multiline strings](/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings)" section to define multi-line environment variables.
```